contrib/babelfishpg_tsql/src/catalog.c

```diff
@@ -1455,6 +1455,11 @@ babelfish_inconsistent_metadata(PG_FUNCTION_ARGS)
 
 	return_consistency = PG_GETARG_BOOL(0);
 
+
+	if(!metadata_inconsistency_check_enabled())
+			return true;
```

Comment:
This is a c implementation for a stored procedure, and the return type is Datum. It should be `PG_RETURN_NULL();`

Reply:
Yes will delete that.

Reply:
Yes will delete that @xhfanhe

Reply:
this should be deleted, and keep line 1449 only.